boards/shields/xenvm_dom0/Kconfig.defconfig

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2023 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_XENVM_DOM0
+
+config NUM_IRQS
```

Comment:
Is there reason why you can't use value from original SoC/Board?

Reply:
different number of IRQ for h3ulcb and XENVM boards, e.g. 240 vs 500